google/ggrc-core
# [5OBJ] QUICK-FIX Add CycleTaskable mixin for Contracts & Policies

# Dependencies

_**This PR should be merged into [GGRC-7200](https://github.com/google/ggrc-core/tree/GGRC-7200) branch which is the common one for all 5OBJ. PRs**_

# Issue description

`CycleTaskable` mixin was missed during movement of Contracts & Policies from `Directives` into separate classes.

# Steps to test the changes

1. Login in GGRC;
2. Open some policy object;
3. Press on three dots menu, select "Create task", and save it.
4. Make sure there is no HTTP errors.

# Solution description

Add `CycleTaskable` to base classes for `Contract` & `Policy` objects.

# Sanity checklist


# PR Review checklist
